compiler/ras/Debug.cpp

```diff
@@ -3912,6 +3912,14 @@ TR_Debug::getRuntimeHelperName(int32_t index)
          case TR_stackOverflow:             return "jitStackOverflow";
          case TR_reportMethodEnter:         return "jitReportMethodEnter";
          case TR_reportStaticMethodEnter:   return "jitReportStaticMethodEnter";
+         case TR_jitReportInstanceFieldRead: return "jitReportInstanceField";
```

Comment:
`"jitReportInstanceFieldRead"`

Reply:
Thanks for pointing this out. Changed according to the suggestion.